src/components/LoanCards/LendHomepageLoanCard.vue

```diff
@@ -248,13 +245,16 @@ $card-height: rem-calc(388);
 
 		.action-button {
 			margin: 0;
-			font-size: rem-calc(20);
 			padding: 0.75rem 1rem;
 		}
+
+		.action-button:not(.loan-funded-text):not(.loan-expired-text):not(.loan-selected-text) {
```

Comment:
Technically, these states should never happen, but removed the font size for them in case they do